coderd/database/queries/groups.sql

```diff
@@ -52,6 +52,10 @@ WHERE
 						)
 				ELSE true
 		END
+		AND CASE WHEN array_length(@group_names :: text[], 1) > 0  THEN
+					 groups.name = ANY(@group_names)
+			ELSE true
+		END
```

Comment:
```suggestion
		AND CASE WHEN array_length(@group_names :: text[], 1) > 0  THEN
				groups.name = ANY(@group_names)
			ELSE true
		END
```
nit; indentation is a little weird